modules/hm/options/security.nix

```diff
@@ -25,6 +25,16 @@ in {
       '';
     };
 
+    wrapWithProxychains = mkOption {
+      type = types.bool;
+      default = false;
+      example = true;
+      description = ''
+        Wrap schizofox desktop entry with proxychains-ng.
```

Comment:
add a space inbetween those two lines, or in generated documentation they will appear on the same line